src/game/default/g_types.h

```diff
@@ -817,6 +825,7 @@ typedef struct {
 
 	const g_item_t *item; // for bonus items
 
+	_Bool held_grenade;	// is this entity a nade that is being held currently?
```

Comment:
currently all those states are used, maybe I can work it a different way to slim it down, later...